doc/run.md

```diff
@@ -6,54 +6,125 @@ This document provides instructions on how to run Nanvix.
 
 ## Table of Contents
 
-- [Running Nanvix Through the Build System](#running-nanvix-through-the-build-system)
-- [Running Nanvix in the MicroVM (Hyperlight and MicroVM Machines Only)](#running-nanvix-in-the-microvm-hyperlight-and-microvm-machines-only)
+- [Running Nanvix with `nanvixd` (Preferred Method)](#running-nanvix-with-nanvixd-preferred-method)
+  - [Step 1: Run `nanvixd`](#step-1-run-nanvixd)
+  - [Step 2: Run an Application](#step-2-run-an-application)
+- [Running Nanvix Components Manually (Hyperlight and MicroVM Machines Only)](#running-nanvix-components-manually-hyperlight-and-microvm-machines-only)
   - [Step 1: Run the Linux Daemon](#step-1-run-the-linux-daemon)
   - [Step 2: Run the MicroVM](#step-2-run-the-microvm)
   - [Enabling Logging (Optional)](#enabling-logging-optional)
   - [Redirecting Standard Error (Optional)](#redirecting-standard-error-optional)
-- [Running Nanvix with `nanvixd` (MicroVM and Hyperlight Machines Only)](#running-nanvix-with-nanvixd-microvm-and-hyperlight-machines-only)
-  - [Step 1: Run `nanvixd`](#step-1-run-nanvixd)
-  - [Step 2: Run an Application](#step-2-run-an-application)
+- [Running Nanvix Through the Build System](#running-nanvix-through-the-build-system)
 
-## Running Nanvix Through the Build System
+## Running Nanvix with `nanvixd` (Preferred Method)
 
-> ℹ️ This runs Nanvix with the default build parameters. Check the [build.md](build.md) document for more information on how to change default build parameters.
+Nanvixd is a utility script that manages the deployment of User VMs in Nanvix, and their corresponding linuxd instances.
 
-To run Nanvix through the build system, simply execute:
+Nanvixd exposes a unified RESTful API to interact with your deployment. To follow this guide we assume you have `jq` installed.
+
+### Step 1: Run `nanvixd`
+
+Open a terminal and run `nanvixd`:
 
 ```bash
-make run
+./bin/nanvixd.elf -http-addr 127.0.0.1:8080 [-tmp-dir <tmp_dir>]
```

Comment:
IMO we don't need to list temporary parameters here. People looking for these docks are looking for copy-paste instructions.

Reply:
Agree.